### ISSUE TITLE
TypeError: install() got an unexpected keyword argument 'prune'

### ISSUE THREAD

**Issue by parente** [conda/conda#2749]
Seems to be a new bug with:

```
    conda:       4.0.6-py35_0 --> 4.1.2-py35_0
    conda-env:   2.4.5-py35_0 --> 2.5.0-py35_0
```

Log with steps I took running against the environment.yml in https://github.com/jupyter/kernel_gateway_demos/tree/master/cf_demo embedded:

```
~/p/j/k/cf_demo ❯❯❯ conda update --yes conda                                
Fetching package metadata: ....
.Solving package specifications: .........

Package plan for installation in environment /Users/parente/miniconda3:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-env-2.5.0            |           py35_0          29 KB
    ruamel_yaml-0.11.7         |           py35_0         165 KB
    conda-4.1.2                |           py35_0         202 KB
    ------------------------------------------------------------
                                           Total:         396 KB

The following NEW packages will be INSTALLED:

    ruamel_yaml: 0.11.7-py35_0

The following packages will be UPDATED:

    conda:       4.0.6-py35_0 --> 4.1.2-py35_0
    conda-env:   2.4.5-py35_0 --> 2.5.0-py35_0

Fetching packages ...
conda-env-2.5. 100% |#########################################################| Time: 0:00:00 436.02 kB/s
ruamel_yaml-0. 100% |#########################################################| Time: 0:00:00 830.63 kB/s
conda-4.1.2-py 100% |#########################################################| Time: 0:00:00 763.66 kB/s
Extracting packages ...
[      COMPLETE      ]|############################################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|############################################################################| 100%
Linking packages ...
[      COMPLETE      ]|############################################################################| 100%
~/p/j/k/cf_demo ❯❯❯ conda install --yes --quiet pip                
Fetching package metadata .......
Solving package specifications: ..........

Package plan for installation in environment /Users/parente/miniconda3:

The following packages will be UPDATED:

    pip: 8.1.1-py35_1 --> 8.1.2-py35_0

~/p/j/k/cf_demo ❯❯❯ conda env update  -n dep_env -f "environment.yml"     
Fetching package metadata .......
Solving package specifications: ..........
Fetching packages ...
decorator-4.0. 100% |#########################################################| Time: 0:00:00 468.69 kB/s
jsonschema-2.5 100% |#########################################################| Time: 0:00:00 530.05 kB/s
markupsafe-0.2 100% |#########################################################| Time: 0:00:00 720.73 kB/s
numpy-1.10.4-p 100% |#########################################################| Time: 0:00:00   6.66 MB/s
ptyprocess-0.5 100% |#########################################################| Time: 0:00:00 737.51 kB/s
pytz-2016.4-py 100% |#########################################################| Time: 0:00:00   1.07 MB/s
pyzmq-15.2.0-p 100% |#########################################################| Time: 0:00:00   2.18 MB/s
simplegeneric- 100% |#########################################################| Time: 0:00:00   7.02 MB/s
tornado-4.3-py 100% |#########################################################| Time: 0:00:00   2.38 MB/s
jinja2-2.8-py3 100% |#########################################################| Time: 0:00:00   1.90 MB/s
pickleshare-0. 100% |#########################################################| Time: 0:00:00   8.58 MB/s
python-dateuti 100% |#########################################################| Time: 0:00:01 217.09 kB/s
terminado-0.6- 100% |#########################################################| Time: 0:00:00 666.59 kB/s
ipython-4.2.0- 100% |#########################################################| Time: 0:00:00   2.97 MB/s
pandas-0.17.1- 100% |#########################################################| Time: 0:00:04   1.45 MB/s
notebook-4.1.0 100% |#########################################################| Time: 0:00:00   7.72 MB/s
Extracting packages ...
[      COMPLETE      ]|############################################################################| 100%
Linking packages ...
[      COMPLETE      ]|############################################################################| 100%
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/Users/parente/miniconda3/bin/conda-env", line 6, in <module>
    sys.exit(conda_env.cli.main.main())
  File "/Users/parente/miniconda3/lib/python3.5/site-packages/conda_env/cli/main.py", line 68, in main
    return args_func(args, parser)
  File "/Users/parente/miniconda3/lib/python3.5/site-packages/conda/cli/main.py", line 127, in args_func
    args.func(args, p)
  File "/Users/parente/miniconda3/lib/python3.5/site-packages/conda_env/cli/main_update.py", line 118, in execute
    installer.install(prefix, specs, args, env, prune=args.prune)
TypeError: install() got an unexpected keyword argument 'prune'
```

**Comment by kalefranz** [comment]
conda update conda-env

Fixed in 2.5.1

Duplicate of https://github.com/conda/conda-env/issues/252

**Comment by parente** [comment]
@kalefranz Thanks!
